src/fontra/views/editor/edit-tools-shape-ellipse.js

```diff
@@ -0,0 +1,31 @@
+import * as rectangle from "../core/rectangle.js";
+import { range } from "../core/utils.js";
+import { ShapeToolRect } from "./edit-tools-shape.js";
+
+const bezierArcMagic = 0.5522847498; // constant for drawing circular arcs w/ Beziers
```

Comment:
Please move all relevant content the edit-tools-shape.js module. We don't need a separate module for this.

Reply:
I removed unnecessary imports.

Reply:
That's not what I meant. The module edit-tools-shape-ellipse.js should not exist, and its contents should be part of edit-tools-shape.js.

Reply:
OK, did it.